apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 90 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27156&commitId=96245fa987d5fe2734794e1786a91aff5e109ff7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1802 screenshots unchanged, 0 screenshots with different dimensions and 90 screenshots with visible difference.


  <details><summary><b>unknown</b> 90 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar converged.Determinate - Dark Mode.chromium.png|4692| Changed|
|ProgressBar converged.Determinate - High Contrast.chromium.png|4961| Changed|
|ProgressBar converged.Determinate - RTL.chromium.png|2818| Changed|
|ProgressBar converged.Determinate with thickness large.chromium.png|5632| Changed|
|ProgressBar converged.Determinate.chromium.png|2818| Changed|
|ProgressBar converged.Error.chromium.png|2671| Changed|
|ProgressBar converged.Indeterminate - Dark Mode.chromium.png|5595| Changed|
|ProgressBar converged.Indeterminate - High Contrast.chromium.png|6694| Changed|
|ProgressBar converged.Indeterminate - RTL.chromium.png|1429| Changed|
|ProgressBar converged.Indeterminate with thickness large.chromium.png|2857| Changed|
|ProgressBar converged.Indeterminate.chromium.png|1429| Changed|
|ProgressBar converged.Success.chromium.png|2967| Changed|
|ProgressBar converged.Warning.chromium.png|2664| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|61595| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|94892| Changed|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|75313| Changed|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|116025| Changed|
|Spinner converged.Inverted with Huge.chromium.png|75313| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|59241| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|91265| Changed|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|95540| Changed|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|147186| Changed|
|Spinner converged.Inverted with Label Above.chromium.png|95540| Changed|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|59241| Changed|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|91265| Changed|
|Spinner converged.Inverted with Label After.chromium.png|59241| Changed|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|59241| Changed|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|91265| Changed|
|Spinner converged.Inverted with Label Before.chromium.png|59241| Changed|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|95540| Changed|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|147186| Changed|
|Spinner converged.Inverted with Label Below.chromium.png|95540| Changed|
|Spinner converged.Inverted with Label.chromium.png|59241| Changed|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|70761| Changed|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|109012| Changed|
|Spinner converged.Inverted with Size Extra Large.chromium.png|70761| Changed|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|52347| Changed|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|80644| Changed|
|Spinner converged.Inverted with Size Extra Small.chromium.png|52347| Changed|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|66189| Changed|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|101968| Changed|
|Spinner converged.Inverted with Size Large.chromium.png|66189| Changed|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|61595| Changed|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|94892| Changed|
|Spinner converged.Inverted with Size Medium.chromium.png|61595| Changed|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|56981| Changed|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|87784| Changed|
|Spinner converged.Inverted with Size Small.chromium.png|56981| Changed|
|Spinner converged.Inverted.chromium.png|61595| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|49540| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|59032| Changed|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|67678| Changed|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|80643| Changed|
|Spinner converged.Primary with Huge.chromium.png|664| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|47667| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|56800| Changed|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|94429| Changed|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|112521| Changed|
|Spinner converged.Primary with Label Above.chromium.png|888| Changed|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|47667| Changed|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|56800| Changed|
|Spinner converged.Primary with Label After - RTL.chromium.png|888| Changed|
|Spinner converged.Primary with Label After.chromium.png|888| Changed|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|47667| Changed|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|56800| Changed|
|Spinner converged.Primary with Label Before - RTL.chromium.png|888| Changed|
|Spinner converged.Primary with Label Before.chromium.png|888| Changed|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|94429| Changed|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|112521| Changed|
|Spinner converged.Primary with Label Below.chromium.png|888| Changed|
|Spinner converged.Primary with Label.chromium.png|888| Changed|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|61658| Changed|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|73471| Changed|
|Spinner converged.Primary with Size Extra Large.chromium.png|464| Changed|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|37316| Changed|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|44465| Changed|
|Spinner converged.Primary with Size Extra Small.chromium.png|193| Changed|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|55613| Changed|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|66267| Changed|
|Spinner converged.Primary with Size Large.chromium.png|413| Changed|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|49540| Changed|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|59032| Changed|
|Spinner converged.Primary with Size Medium.chromium.png|363| Changed|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|43442| Changed|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|51764| Changed|
|Spinner converged.Primary with Size Small.chromium.png|221| Changed|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|47691| Changed|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|73472| Changed|
|Spinner converged.Primary with Size Tiny.chromium.png|47691| Changed|
|Spinner converged.Primary.chromium.png|363| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>